lib/src/protocol_instance.rs

```diff
@@ -140,7 +153,7 @@ impl ProtocolInstance {
             prover: input.taiko.prover_data.prover,
             chain_id: input.chain_spec.chain_id,
             verifier_address,
-            proof_of_equivalence,
+            // proof_of_equivalence,
```

Comment:
remove

Reply:
still a little bit concerned about using proof of equivalence without verification.

Reply:
The proof of equivalence is now verified inside the zkVM itself, so no more 50k gas onchain or smart contract changes necessary to support it. So whatever is the minimal number of cycles to prove now, which surprisingly is proof of equivalence inside the zkVM (at least for now).